src/Adapter/Driver/Pdo/Connection.php

```diff
@@ -237,6 +240,9 @@ public function connect()
                     if (isset($charset) && $pdoDriver != 'pgsql') {
                         $dsn[] = "charset={$charset}";
                     }
+                    if (isset($unix_socket)) {
```

Comment:
I think it would be best to add the if else statement since the manual does explicitly point it out for PDO.